src/main/pg/alt_hold.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <stdint.h>
+
+#include "pg/pg.h"
+
+typedef struct altholdConfig_s {
+    uint8_t alt_hold_pid_p;
+    uint8_t alt_hold_pid_i;
+    uint8_t alt_hold_pid_d;
+    uint16_t alt_hold_throttle_min;
+    uint16_t alt_hold_throttle_max;
+    uint8_t alt_hold_target_adjust_rate;
```

Comment:
```suggestion
    uint8_t alt_hold_target_adjust_rate;
    uint16_t alt_hold_throttle_min;
    uint16_t alt_hold_throttle_max;
```
as I know it is better to align types in structures to optimize memory access. If I'm right this should save 1 byte

Reply:
Fascinating!  I never knew that elements of a struct should be grouped.  Seemed to save 2 bytes :-)